lib/src/components/modal_main_page.dart

```diff
@@ -77,6 +70,14 @@ class _ModalMainPageState extends State<ModalMainPage> {
       ),
     );
   }
+
+  _Segment listSegment() => _Segment(
```

Comment:
@elliotsayes Can you please convert these to StatelessWidget which is preferable to functions? I know it's a bit of an overkill but rather keep it in line with best practices.

Reply:
I didn't realise this, thanks